examples/md-flexible/src/ParallelVtkWriter.cpp

```diff
@@ -0,0 +1,201 @@
+  timestepFile << "  <UnstructuredGrid>\n";
+  timestepFile << "    <Piece NumberOfCells=\"0\" NumberOfPoints=\"" << numberOfParticles << "\">\n";
+  timestepFile << "      <PointData>\n";
+
+  // print velocities
+  timestepFile
+      << "        <DataArray Name=\"velocities\" NumberOfComponents=\"3\" format=\"ascii\" type=\"Float32\">\n";
+  for (auto particle = autoPasContainer.begin(autopas::IteratorBehavior::owned); particle.isValid(); ++particle) {
+    auto v = particle->getV();
+    timestepFile << "        " << v[0] << " " << v[1] << " " << v[2] << "\n";
+  }
+  timestepFile << "        </DataArray>\n";
+
+  // print forces
+  timestepFile << "        <DataArray Name=\"forces\" NumberOfComponents=\"3\" format=\"ascii\" type=\"Float32\">\n";
+  for (auto particle = autoPasContainer.begin(autopas::IteratorBehavior::owned); particle.isValid(); ++particle) {
+    auto f = particle->getF();
+    timestepFile << "        " << f[0] << " " << f[1] << " " << f[2] << "\n";
+  }
+  timestepFile << "        </DataArray>\n";
+
+  // print type ids
+  timestepFile << "        <DataArray Name=\"typeIds\" NumberOfComponents=\"1\" format=\"ascii\" type=\"Float32\">\n";
+  for (auto particle = autoPasContainer.begin(autopas::IteratorBehavior::owned); particle.isValid(); ++particle) {
+    timestepFile << "        " << particle->getTypeId() << "\n";
+  }
+  timestepFile << "        </DataArray>\n";
+
+  // print ids
+  timestepFile << "        <DataArray Name=\"ids\" NumberOfComponents=\"1\" format=\"ascii\" type=\"Float32\">\n";
+  for (auto particle = autoPasContainer.begin(autopas::IteratorBehavior::owned); particle.isValid(); ++particle) {
+    timestepFile << "        " << particle->getID() << "\n";
+    ;
+  }
+  timestepFile << "        </DataArray>\n";
+
+  timestepFile << "      </PointData>\n";
+  timestepFile << "      <CellData/>\n";
+  timestepFile << "      <Points>\n";
+
+  // print positions
+  timestepFile << "        <DataArray Name=\"position\" NumberOfComponents=\"3\" format=\"ascii\" type=\"Float32\">\n";
+  for (auto particle = autoPasContainer.begin(autopas::IteratorBehavior::owned); particle.isValid(); ++particle) {
+    auto pos = particle->getR();
+    timestepFile << "        " << pos[0] << " " << pos[1] << " " << pos[2] << "\n";
+  }
+  timestepFile << "        </DataArray>\n";
+
+  timestepFile << "      </Points>\n";
+  timestepFile << "      <Cells>\n";
+  timestepFile << "        <DataArray Name=\"types\" NumberOfComponents=\"0\" format=\"ascii\" type=\"Float32\"/>\n";
+  timestepFile << "      </Cells>\n";
+  timestepFile << "    </Piece>\n";
+  timestepFile << "  </UnstructuredGrid>\n";
+  timestepFile << "</VTKFile>\n";
+
+  timestepFile.close();
+}
+
+void ParallelVtkWriter::tryCreateSessionAndDataFolders(const std::string &name, std::string location) {
+  time_t rawTime;
+  time(&rawTime);
+
+  struct tm timeInformation;
+  gmtime_r(&rawTime, &timeInformation);
```

Comment:
Why did you change this from `localtime`? Local timezones seem a bit less confusing than GMT.

Reply:
Using localtime was considered dangerous by one of the bots. Although this introduced i minor bug (time is 2 hours off) I am planning to find a way to generate the local time without using localtime.